helm/chart/maesh/values.yaml

```diff
@@ -68,7 +68,9 @@ metrics:
   prometheus:
     enabled: true
 
-smi: false
+smi:
+  deploy: true
+  enable: false
```

Comment:
Won't you need to change the controller args for this? since now `.Values.smi` is not a bool anymore?

Reply:
Solved.